lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
Great! Thanks for the feedbacks @albinekb 😄 

So, if @rauchg approves, I remove all this part of executability check and I keep only the base behaviour we talked about:
- `cd ${path}\n` if it's a folder
- `path` without return for any file